packages/gestalt-codemods/34.0.0/__testfixtures__/ternary-color.input.js

```diff
@@ -3,6 +3,7 @@ import { Toast } from 'gestalt';
 
 export default function TestComponent() {
   return (
+    // eslint-disable-next-line no-constant-condition
```

Comment:
these were unintentional — didn't branch off master, but landed that other PR first